src/main/java/com/pollapp/controller/CategoryController.java

```diff
@@ -50,12 +51,28 @@ public Category deleteCategory(@PathVariable long categoryId) {
         // TODO
         return null;
     }
-
-    @GetMapping("/{categoryId}/polls")
-    public List<PollResponse> getPollsByCategory(@PathVariable int categoryId) {
+    
+    @GetMapping("/{categoryId}/all")
+    public List<PollResponse> getAllPollsByCategory(@PathVariable int categoryId) {
         List<PollResponse> response = new ArrayList<>();
         pollRepository.findAllByCategoriesId(categoryId).forEach(poll ->
                 response.add(new PollResponse(poll, pollProcess.process(poll))));
         return response;
     }
+
+    @GetMapping("/{categoryId}/closed")
+    public List<PollResponse> getClosedPollsByCategory(@PathVariable int categoryId) {
+        List<PollResponse> response = new ArrayList<>();
+        pollRepository.findAllByClosedBeforeOrClosedIsNullAndCategoriesId(Calendar.getInstance(), categoryId).forEach(poll ->
+                response.add(new PollResponse(poll, pollProcess.process(poll))));
+        return response;
+    }
+    
+    @GetMapping("/{categoryId}/ongoing")
+    public List<PollResponse> getOngoingPollsByCategory(@PathVariable int categoryId) {
```

Comment:
/{categoryId}/polls/ongoing or /{categoryId}/polls with request param like closed=false or open=true is more appropriate for rest api